tests/to_pyg_test.py

```diff
@@ -0,0 +1,82 @@
+
+from concurrent.futures.thread import ThreadPoolExecutor
+
+import networkx as nx
+import pytest
+
+import programl as pg
+from torch_geometric.data import HeteroData
+from tests.test_main import main
+
+pytest_plugins = ["tests.plugins.llvm_program_graph"]
+
+
+@pytest.fixture(scope="session")
+def graph() -> pg.ProgramGraph:
+    return pg.from_cpp("int A() { return 0; }")
+
+def assert_equal_graphs(
+    graph1: HeteroData,
+    graph2: HeteroData
+):
+    assert graph1['nodes']['text'] == graph2['nodes']['text']
+
+    assert graph1['nodes', 'control', 'nodes'].edge_index.equal(graph2['nodes', 'control', 'nodes'].edge_index)
+    assert graph1['nodes', 'data', 'nodes'].edge_index.equal(graph2['nodes', 'data', 'nodes'].edge_index)
+    assert graph1['nodes', 'call', 'nodes'].edge_index.equal(graph2['nodes', 'call', 'nodes'].edge_index)
+    assert graph1['nodes', 'type', 'nodes'].edge_index.equal(graph2['nodes', 'type', 'nodes'].edge_index)
```

Comment:
Great! Since you're testing for equality, you should probably test for inequality too by adding a unit test to cover the case where say `pg.from_cpp("int A(){return 0;}") != pg.from_cpp("int B(){return 1;}")`